src/group_theory/index.lean

```diff
@@ -147,6 +147,9 @@ by rw [relindex, subgroup_of_bot_eq_top, index_top]
 @[simp, to_additive] lemma relindex_self : H.relindex H = 1 :=
 by rw [relindex, subgroup_of_self, index_top]
 
+@[simp, to_additive] lemma card_mul_index : nat.card H * H.index = nat.card G :=
```

Comment:
```suggestion
@[simp, to_additive card_mul_index] 
lemma card_mul_index : nat.card H * H.index = nat.card G :=
```
See #9945 , without this addition the additive lemma is called `add_subgroup.card_add_index`.